### ISSUE TITLE
Ability to filter Quest and PCVR players on the Server/Relay 

### ISSUE THREAD

**Issue by vibinator** [EchoTools/EchoRelay#24]
Ability to filter Quest and PCVR players on the Server.  (To block Quest players)

(Quest Players will overload servers)

**Comment by thesprockee** [comment]
Hardware info is sent in one of the first few messages. It will be straight-forward to block non-pcvr.

**Comment by MathieDev** [comment]
Quest isn't even done yet though..

**Comment by vibinator** [comment]
> Quest isn't even done yet though..

spoke too soon

**Comment by tekemperor** [comment]
Is this something we want to do? Being able to identify it is fine, but what is the objective with the filtering? Does the quest cause different load on the system, or is this more about the number of players and/or some preconceived notions of player behavior? I think opt-in communities might be a more pragmatic approach to the general problem of trying to decide who is welcome on a given server.

**Comment by vibinator** [comment]
Yes, this isn't an issue now due to having server capacity, but i would
like to have the ability to know how many PCVR and Quest Standalone players
we have :)
